centos/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM centos
+RUN  echo $'[google-cloud-sdk]\n\
+name=Google Cloud SDK\n\
+baseurl=https://packages.cloud.google.com/yum/repos/cloud-sdk-el7-x86_64\n\
+enabled=1\n\
+gpgcheck=1\n\
+repo_gpgcheck=1\n\
+gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg\n\
+       https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg'\
+        >> /etc/yum.repos.d/google-cloud-sdk.repo && \
+    yum install -y google-cloud-sdk which && \
+    gcloud config set core/disable_usage_reporting true && \
+    gcloud config set component_manager/disable_update_check true
+VOLUME ["/.config"]
```

Comment:
yes! (i thought i added that in but must've missed it).

next push will have 
FROM centos:centos7
VOLUME ["/root/.config"]